### PR TITLE
Fixed padding issues, animation issues

### PR DESCRIPTION
**Pull request by AutinMitra**
Add proper padding for FAQ section header, added space for the FAQ footer. Fixed animation for teams, and made team members respond a bit better on mobile (centered)

### CODE REVIEW

**Comment by AutinMitra** [comment]
Added some padding to one of the divs, now there is whitespace on side of page. Fix incoming.

**Comment by AutinMitra** [comment]
Added some rounding to team members and the buttons, and fixed that whitespace issue. Feel free to edit.

File: src/index.css [hunk 1]
```diff
@@ -1802,6 +1802,7 @@ a.altbutton {
 }
 
 a.flatbutton {
+  border-radius: 1rem;
```
Comment (sumanthratna) [inline src/index.css]: I think it looks better without; remove this line.

**Review by sumanthratna — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 1 above)
Comment (AutinMitra) [reply src/index.css]: Removed in last commit

**Comment by AutinMitra** [comment]
Added the transitions for FAQ, should be good

File: src/MainPage/Questions/FAQuestion/index.jsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ class FAQuestion extends Component {
         <div
           className="answer"
           style={{
-            height: this.state.showAnswer ? "100%" : 0
+            "max-height": this.state.showAnswer ? "500px" : 0
```
Comment (sumanthratna) [inline src/MainPage/Questions/FAQuestion/index.jsx]: Is there a reason you chose to limit the height to 500px? The answer to "Can I be a mentor, judge, or volunteer?" is cut off.

**Review by sumanthratna — CHANGES_REQUESTED** [review]

File: src/MainPage/Questions/FAQuestion/index.jsx (hunk 2 above)
Comment (AutinMitra) [reply src/MainPage/Questions/FAQuestion/index.jsx]: I thought 500 was adequate, my bad will boost to 600px

Height transitions don't work with ``auto`` or ``100%``, so an alternative is to instead transition max-height between 0 and a set value, so the div will reach "auto" size, and still retain proper animation.

Comment (AutinMitra) [reply src/MainPage/Questions/FAQuestion/index.jsx]: Fixed the issue

**Pull request merged**